<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>676</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1119</width>
    <height>676</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1119</width>
    <height>676</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(171, 171, 171);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>420</y>
      <width>421</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="hand_player">
     <item>
      <widget class="QLabel" name="handPlayer1">
       <property name="minimumSize">
        <size>
         <width>136</width>
         <height>189</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>136</width>
         <height>189</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 1px;
border-color: black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="handPlayer2">
       <property name="minimumSize">
        <size>
         <width>136</width>
         <height>189</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>136</width>
         <height>189</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 1px;
border-color: black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="handPlayer3">
       <property name="minimumSize">
        <size>
         <width>136</width>
         <height>189</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>136</width>
         <height>189</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 1px;
border-color: black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>470</y>
      <width>101</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="buttons">
     <item>
      <widget class="QPushButton" name="pushButton_drawMore">
       <property name="text">
        <string>Взять еще</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_enough">
       <property name="text">
        <string>Оставить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_surrender">
       <property name="text">
        <string>Сдаться</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_double">
       <property name="text">
        <string>Удвоить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_split">
       <property name="text">
        <string>Разделить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>422</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="hand_diller">
     <item>
      <widget class="QLabel" name="handDealer1">
       <property name="minimumSize">
        <size>
         <width>136</width>
         <height>189</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>136</width>
         <height>189</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 1px;
border-color: black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="handDealer2">
       <property name="minimumSize">
        <size>
         <width>136</width>
         <height>189</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>136</width>
         <height>189</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 1px;
border-color: black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="handDealer3">
       <property name="minimumSize">
        <size>
         <width>136</width>
         <height>189</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>136</width>
         <height>189</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 1px;
border-color: black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>329</y>
      <width>421</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLabel" name="CardValuePlayer2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="CardValuePlayer1">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="CardValuePlayer3">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="CardValuePlayerTotal">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>421</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="CardValueDealer1">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="CardValueDealer3">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="CardValueDealer2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="CardValueDealerTotal">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="output">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>256</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_newgame">
       <property name="text">
        <string>Новая игра</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Ставка</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Баланс</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="cash">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1119</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
